app/controllers/carts_controller.rb

```diff
@@ -14,4 +13,17 @@ def create
     flash[:notice] = "You now have #{pluralize(@basket.count_of(item.id), item.title)}."
     redirect_to bike_shop_path
   end
+
+  def update
+    @basket.update_item(params[:cart][:item], params[:cart][:quantity])
+    flash[:notice] = "Cart Updated!"
+    redirect_to cart_path
+  end
+
+  def destroy
+    @basket.remove(params[:item])
+    item = Item.find(params[:item].to_i)
+    flash[:notice] = "Successfully removed <a href=\"#{item_path(item)}\">#{item.title}</a> from your cart."
```

Comment:
This is displaying <a href > instead of converting it to html. Unsure why.
Any suggestions on how to fix this?

Reply:
Fixed with sanatize in layouts

Reply:
Looks good to me